src/lib.rs

```diff
@@ -220,7 +247,99 @@ impl Api {
         params.add_get_opt("timeout", timeout);
 
         // Execute request
-        self.send_request("getUpdates", params)
+        self.send_request("getUpdates", params, RequestType::Post)
+    }
+
+    /// Corresponds to the `sendPhoto` method of the API.
+    pub fn send_photo(&self, chat_id: Integer, path: String, caption: Option<String>, reply_to_message_id: Option<Integer>, reply_markup: Option<ReplyMarkup>) -> Result<Message> {
```

Comment:
If you'd wrap those long line, I'd be delighted ;)

How to properly wrap in Rust is still discussed, but I like this method:
```rust
    pub fn send_photo(&self, chat_id: Integer, 
                      path: String, 
                      caption: Option<String>, 
                      reply_to_message_id: Option<Integer>, 
                      reply_markup: Option<ReplyMarkup>) 
        -> Result<Message> 
    {
```